### PR TITLE
CI-81: Content of the Access Site page is shown incorrectly after opening app on Heroku

### PR DESCRIPTION
**Pull request by VitaliiShevchenko**
[CI-81](https://cloverpop-internship.atlassian.net/browse/CI-81)

[CI-81]: https://cloverpop-internship.atlassian.net/browse/CI-81?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -90,6 +90,80 @@ body {
...
+
+//new.html.erb
+.w746-h59 {
+  width: 746px;
+  height: 59px;
+}
+.mt125 {
+  margin-top: 125px;
+}
+.mt--17 {
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:109]: I would recommend to use something like 'mt-negative-17'

File: app/assets/stylesheets/main.scss [hunk 2]
```diff
@@ -90,6 +90,80 @@ body {
...
+  margin-top:-17px !important;
+}
+.mb115{
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:112]: the same

File: app/assets/stylesheets/main.scss [hunk 3]
```diff
@@ -90,6 +90,80 @@ body {
...
+  margin-bottom: 115px;
+}
+.w588-h59 {
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:115]: the same

File: app/assets/stylesheets/main.scss [hunk 4]
```diff
@@ -90,6 +90,80 @@ body {
...
+  width:588px;
+  height:59px;
+}
+.w660 {
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:119]: maybe - 'w-660' ?

File: app/assets/stylesheets/main.scss [hunk 5]
```diff
@@ -90,6 +90,80 @@ body {
...
+  width:660px;
+}
+.mt162{
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:122]: the same

File: app/assets/stylesheets/main.scss [hunk 6]
```diff
@@ -90,6 +90,80 @@ body {
...
+  top: 767px;
+  min-width:650px;
+}
+.h0{
+  height: 0;
+}
+.h52{
+  height: 52px !important;
+}
+
```
Comment (SokalMet) [inline app/assets/stylesheets/main.scss:166]: all those names you should update ^

**Review by SokalMet — CHANGES_REQUESTED** [review]

**Review by SerhiiBorozenets — APPROVED** [review]

**Pull request merged**